server/executor/trigger/http.go

```diff
@@ -81,7 +82,17 @@ func (te *httpTriggerer) Trigger(ctx context.Context, test model.Test, opts *Tri
 	if tReq.Body != "" {
 		body = bytes.NewBufferString(tReq.Body)
 	}
-	req, err := http.NewRequestWithContext(ctx, strings.ToUpper(string(tReq.Method)), tReq.URL, body)
+
+	parsedUrl, err := url.Parse(tReq.URL)
+	if err != nil {
+		return response, err
+	}
+
+	if parsedUrl.Scheme == "" {
+		parsedUrl.Scheme = "http"
```

Comment:
Should we have a constant for this?

Reply:
I don't think a constant is needed here. it's not a value that will be referenced any other place, it's just a default used in this only context.